### PR TITLE
add space between header, title and date picker input

### PR DESCRIPTION
**Pull request by amanbhatt2022**
#1858 
### Description
I Add space in between header, Title and date picker input by using <pre> tags.

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by P4NK4J** [comment]
@amanbhatt2022 There are some merge conflicts in the file, please check.

**Review by P4NK4J — CHANGES_REQUESTED** [review]
left some comments, please resolve merge conflicts

File: resources/views/hr/recruitment/reports.blade.php [hunk 1]
```diff
@@ -18,6 +18,8 @@
                 <br>
             </div>
         </div>
+        <pre></pre>
```
Comment (mohitgusain) [inline resources/views/hr/recruitment/reports.blade.php:21]: what is the purpose of adding pre tag ?

File: resources/views/hr/recruitment/reports.blade.php [hunk 2]
```diff
@@ -5,8 +5,8 @@
 <div class="chart-container" align="center">
     <div class="bar-chart-container">
         <div class="d-flex justify-content-between">
-            <div class="my-2">
-                <h2>Analytics </h2>
+             <div>
+                <h2>Analytics t</h2>
```
Comment (mohitgusain) [inline resources/views/hr/recruitment/reports.blade.php:9]: check this line why t in the end ?

**Review by mohitgusain — COMMENTED** [review]
@amanbhatt2022 I think your changes are not matching with the issues. Please check once

File: resources/views/hr/recruitment/reports.blade.php (hunk 1 above)
Comment (amanbhatt2022) [reply resources/views/hr/recruitment/reports.blade.php:21]: I use <pre> tag for maintaining space between header and title.

**Pull request closed without merging**